source/extensions/omni.isaac.lab/omni/isaac/lab/assets/articulation/articulation.py

```diff
@@ -539,6 +539,13 @@ def write_joint_limits_to_sim(
             env_ids = env_ids[:, None]
         # set into internal buffers
         self._data.joint_limits[env_ids, joint_ids] = limits
+        # update default joint pos to stay within the new limits
```

Comment:
Just another question here, if someone does this, the default actions used inside the action term for offset would also need to change, right?

Reply:
Yes I think we should rather throw an error here